MarathonRecomp/api/lua/lua.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <Marathon.inl>
+
+namespace lua50
+{
+    typedef xpointer<void> lua_State;
+    typedef be<uint32_t> lua_CFunction;
+
+    struct luaL_reg
+    {
+        xpointer<const char> m_pName;
+        lua_CFunction m_Func;
```

Comment:
Use official Lua variable names.

```suggestion
        xpointer<const char> name;
        lua_CFunction func;
```